practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>"In the movie recommendation system, we have:" -&gt; "In the movie recommendation system:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>resolved in next commit</p>